eng/testing/tests.mobile.targets

```diff
@@ -149,7 +150,11 @@
       MainJS="$(MonoProjectRoot)\wasm\runtime-test.js"
       ExtraAssemblies="@(ExtraAssemblies)"
       FilesToIncludeInFileSystem="@(WasmFilesToIncludeInFileSystem)"
-      AssemblySearchPaths="@(AssemblySearchPaths)" />
+      AssemblySearchPaths="@(AssemblySearchPaths)"
+      AssetSources="@(AssetSources)"
+      Assets="@(Assets)" />
+      <Copy SourceFiles="$(NuGetPackageRoot)\microsoft.netcore.runtime.icu.transport\$(MicrosoftNETCoreRuntimeICUTransportVersion)\runtimes\browser-wasm\native\lib\icudt.dat"
```

Comment:
this can be replaced with `$(PkgMicrosoft_NETCore_Runtime_ICU_Transport)` now that https://github.com/dotnet/runtime/commit/8ed502f7427925b94c26adc479876252538c9147 was merged, you just need to add a PackageReference:

 ```
  <ItemGroup>
    <PackageReference Include="$(MicrosoftNETCoreRuntimeICUTransportPackage)" PrivateAssets="all" Version="$(MicrosoftNETCoreRuntimeICUTransportVersion)" GeneratePathProperty="true" />
  </ItemGroup>
```

Reply:
actually scratch that, we should put the icu data file into the runtime pack

Reply:
how am I supposed to resolve this comment?

Reply:
we don't necessarily need to do it as part of this PR, I'll take a look